src/content-tags-drawer/tags-sidebar/TagsTree.jsx

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
```

Comment:
Missing type checking
```suggestion
// @ts-check
import PropTypes from 'prop-types';
```

Reply:
Updated here https://github.com/openedx/frontend-app-course-authoring/pull/852/commits/1d505dad132b3194ae3326e4b59e01b69672e531